tests/unit/components/CostsPie.spec.ts

```diff
@@ -75,8 +76,8 @@ describe("costs card", () => {
 
   it("should populate the cost data into the chart when the data is loaded after the component is mounted", async () => {
     await mountSuspended(CostsPie, {
-      global: { stubs, plugins: [mockPinia()] },
-      props: { hideTooltips: false, pieSize: 100 },
+      global: { stubs, plugins: [mockPinia({ metadata: mockedMetadata }, true, false)] },
```

Comment:
You don't need to import and pass in the mockedMetadata -- mockPinia already does this by default (if you set  the second parameter to true)